README.md

```diff
@@ -7,7 +7,6 @@
 [![Code Climate](https://img.shields.io/codeclimate/github/18F/openFEC.svg)](https://codeclimate.com/github/18F/openFEC)
 [![Dependencies](https://img.shields.io/gemnasium/18F/openFEC.svg)](https://gemnasium.com/18F/openFEC)
 
-![Swagger validation badge](https://online.swagger.io/validator?url=https://api.open.fec.gov/swagger)
```

Comment:
Currently our swagger spec file fails and doesn't display correctly, so I've removed it until we can figure out what's going on.

Reply:
This hasn't worked in the longest time, I've tried to fix it before and couldn't.  I wonder if this requires an update to the latest version to have it work?

Reply:
When I tried out 3.x it prominently showed a failing status badge on the API docs itself. I think your assumption might be correct, the spec file probably needs some update to have it pass on 3.x.